File: engine/engine-model/src/main/scala/net/server/channel/packet/keymap/RegularKeymapChangePacket.scala
package net.server.channel.packet.keymap

class RegularKeymapChangePacket(private var _available: Boolean, private var _mode: Int, private var _changes: Array[KeyTypeAction]) extends BaseKeymapChangePacket(_available = _available, _mode = _mode) {
  def changes: Array[KeyTypeAction] = _changes
}

File: engine/engine-model/src/main/java/net/server/channel/packet/keymap/RegularKeymapChangePacket.java
package net.server.channel.packet.keymap;

public class RegularKeymapChangePacket extends BaseKeymapChangePacket {
   private final KeyTypeAction[] changes;

   public RegularKeymapChangePacket(Boolean available, Integer mode, KeyTypeAction[] changes) {
      super(available, mode);
      this.changes = changes;
   }

   public KeyTypeAction[] changes() {
      return changes;
   }
}
